ProfileManager.js

```diff
@@ -781,40 +934,79 @@ export default class ProfileManager {
       invocationSigner
     });
 
+    const promise = this._readAgentContent(
+      {edvDocument, zcaps: profileAgent.zcaps});
```

Comment:
Does this fit in 80 chars?

```suggestion
    const promise = this._readAgentContent({edvDocument, zcaps: profileAgent.zcaps});
```

Reply:
No.